compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/codegen/split/constants/JvmArrayTypeConstantsGen.java

```diff
@@ -73,6 +76,8 @@ public JvmArrayTypeConstantsGen(PackageID packageID, BTypeHashComparator bTypeHa
         this.arrayTypeVarMap = new TreeMap<>(bTypeHashComparator);
         this.funcNames = new ArrayList<>();
         this.types = types;
+        arrayTypeVarCount = 0;
+        methodCount = 1;
```

Comment:
Shall we move the initialisation to the declaration? It is preferred due to [this](https://stackoverflow.com/questions/3918578/should-i-initialize-variable-within-constructor-or-outside-constructor). We can change in the other places also. Same can be done for `funcName` array list.